packages/cmrjs/src/cmr-utils.js

```diff
@@ -173,17 +189,18 @@ async function publishUMMGJSON2CMR(cmrFile, cmrClient) {
  * @param {string} creds.username - the CMR username, not used if creds.token is provided
  * @param {string} creds.password - the CMR password, not used if creds.token is provided
  * @param {string} creds.token - the CMR or Launchpad token,
+ * @param {string} cmrRevisionId - Optional CMR Revision ID
  * if not provided, CMR username and password are used to get a cmr token
  */
-async function publish2CMR(cmrPublishObject, creds) {
+async function publish2CMR(cmrPublishObject, creds, cmrRevisionId = undefined) {
```

Comment:
We need unit tests for `publish2CMR` to prove that the `cmrRevisionId` is passed on to `publishECHO10XML2CMR` and `publishUMMGJSON2CMR`

Reply:
[`6638647` (#2079)](https://github.com/nasa/cumulus/pull/2079/commits/663864799422184f64130dadd8e00e8930a2606e)

Reply:
Separately, I think `cmrRevisionId = undefined` is unnecessary. It will be undefined if you don't specify a value at call time